source/_integrations/pi_hole.markdown

```diff
@@ -28,7 +28,7 @@ The Pi-hole integration allows you to retrieve statistics and interact with a
 
 Please note, that during the integration set up, an API key can be provided.
 Providing one, gives access to the Pi-Hole `disable` service and a switch
-to enable/disable Pi-Hole from Home Assistant.
+to enable/disable Pi-Hole from Home Assistant. If you check "Statistics only" in setup then you will not be able to control the Pi-Hole.
```

Comment:
Little fluff :) (telling what it will do, not just what it won't do).

```suggestion
to enable/disable Pi-Hole from Home Assistant. If you check "Statistics only"
during setup, then, you will not be able to control Pi-Hole, but only entities
providing statistical information are provided.
```